lib/services/history/history_manager.ts

```diff
@@ -141,10 +163,20 @@ export class SNHistoryManager extends PureService {
     }
   }
 
-  historyForItem(item: SNItem) {
+  sessionHistoryForItem(item: SNItem) {
     return this.historySession!.historyForItem(item.uuid);
   }
 
+  async fetchHistoryFromServer(item: SNItem) {
```

Comment:
`fetchHistoryFromServer` and `serverHistoryForItem` should be combined into one function called `serverHistoryForItem` which returns an array of revisions.